website/docs/d/org_vdc.html.markdown

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
 
 ## Attribute Reference
 
+~> **Note:** Only part of fields are read if user is Organization administrator. With System Admin user all fields are populated.
```

Comment:
Could you list which fields are skipped with Org admin?

Reply:
I would like to ask the same, but to list it not here, but below. I propose using a flag in the parenthesis to reflect which fields are sys-admin-only. And use this flag across all the data sources where org admin has less visibility.

For example:
```
field_a - some common field ...
field_x - (System Admin) some field visible only to sys admins ...
field_z - some another common field ...
```

@dataclouder  @vbauzysvmware @Didainius can we agree on such notion?

Reply:
Updated

Reply:
The result looks really easy to read, thanks! I'll leave it to @dataclouder to resolve this conversation as he started it.